docsets/Werkzeug/docset.json

```diff
@@ -1,10 +1,22 @@
 {
     "name": "Werkzeug",
-    "version": "0.9.6",
+    "version": "0.16.1",
     "archive": "Werkzeug.tgz",
```

Comment:
You need to update the archive at this path as well.

Reply:
Done. https://github.com/Kapeli/Dash-User-Contributions/pull/2675/commits/ea0a9e474f4969e522905e8b4a369bf27579922e